### PR TITLE
Add chain, stateSchema, setFinalityConfirmation, saveStrategy

### PR DESCRIPTION
**Pull request by smaknsk**
This pull request adds 

- **stateSchema** to allow multiple squids to use the database.
- **saveStrategy** for insert or upsert
- **chain** and **setFinalityConfirmation**  parameters for configuration RPC

### CODE REVIEW

**Comment by smaknsk** [comment]
That pull request adds DB_SCHEMA to make it convenient to share the same database for the application and the squid:
https://github.com/subsquid/squid-sdk/pull/199

File: squid-gen-evm/src/processor.ts [hunk 1]
```diff
@@ -13,8 +13,9 @@ export class ProcessorCodegen {
         private outDir: OutDir,
         private options: {
             archive: SquidArchive
+            chain?: SquidChainRpc
             contracts: SquidContract[]
-            from?: number
```
Comment (smaknsk) [inline squid-gen-evm/src/processor.ts:17]: from not used

Comment (abernatskiy) [reply squid-gen-evm/src/processor.ts:17]: Yep

**Comment by abernatskiy** [comment]
Thank you for your contribution!

**Comment by smaknsk** [comment]
@abernatskiy Could you please release it as a new version?

**Pull request merged**